### PR TITLE
starting docs build-out

### PR DESCRIPTION
**Pull request by dzsquared**
still todo before ready for merge - expanding the tutorial

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +24,45 @@ dotnet tool install -g microsoft.sqlpackage
...
+```
+
+### 🛳️ Publish a SQL project
+
+Publish a SQL project to a database using the SqlPackage `publish` command. Learn more about the `publish` command in the [SqlPackage documentation](https://learn.microsoft.com/sql/tools/sqlpackage/sqlpackage-publish), where additional examples and details on the parameters are available.
+
+```bash
+# example publish from Azure SQL Database using SQL authentication and a connection string
+SqlPackage /Action:Publish /SourceFile:"bin\Debug\AdventureWorksLT.dacpac" \
+    /TargetConnectionString:"Server=tcp:{yourserver}.database.windows.net,1433;Initial Catalog=AdventureWorksLT;Persist Security Info=False;User ID=sqladmin;Password={your_password};MultipleActiveResultSets=False;Encrypt=True;TrustServerCertificate=False;Connection Timeout=30;"
```
Comment (ErikEJ) [inline README.md:63]: Simplify by removing default values:

`Server=tcp:{yourserver}.database.windows.net,1433;Initial Catalog=AdventureWorksLT;User ID=sqladmin;Password={your_password};Connection Timeout=30;`

File: src/Microsoft.Build.Sql/docs/Converting-Existing.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# Converting an existing project to use Microsoft.Build.Sql
+
+## Changes to .sqlproj file
+To convert a database project into SDK-style, edit the .sqlproj file by adding
+```xml
+<Sdk Name="Microsoft.Build.Sql" Version="0.1.9-preview" />
+``` 
+inside `<Project>` tag.
```
Comment (ErikEJ) [inline src/Microsoft.Build.Sql/docs/Converting-Existing.md:8]: inside the

File: src/Microsoft.Build.Sql/docs/Converting-Existing.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+```xml
+<Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+```
+```xml
+<Import Condition="..." Project="...\Microsoft.Data.Tools.Schema.SqlTasks.targets"/>
+```
+
+* Note: This only applies to the auto-generated statements that come from SSDT or ADS. Please keep any custom `<Import>` targets that you may have added to the project file.
+
+## Building .sqlproj
```
Comment (ErikEJ) [inline src/Microsoft.Build.Sql/docs/Converting-Existing.md:28]: This section is quite unclear and referring to concepts not mentioned elsewhere

File: README.md [hunk 4]
```diff
@@ -4,12 +4,22 @@
...
 |[Microsoft.Build.Sql.Templates](https://www.nuget.org/packages/Microsoft.Build.Sql.Templates)|Microsoft.Build.Sql.Templates (preview) is a set of [.NET project templates](https://learn.microsoft.com/dotnet/core/tools/custom-templates) for SQL projects. In preview, [source code](/src/Microsoft.Build.Sql.Templates/) in this repository.|
+|[Microsoft.SqlPackage](https://www.nuget.org/packages/Microsoft.SqlPackage)|Microsoft.SqlPackage is a cross-platform command-line utility for creating and deploying .dacpac and .bacpac packages. SqlPackage can be installed as a *dotnet tool*.|
+|[Microsoft.SqlServer.DacFx](https://www.nuget.org/packages/Microsoft.SqlServer.DacFx)|The Microsoft SQL Server Data-Tier Application Framework (DacFx) is a .NET library which provides application lifecycle services for database development and management for Microsoft SQL Server and Microsoft Azure SQL Databases. Preview versions of DacFx are frequently released to NuGet.|
+|[Microsoft.SqlServer.Dacpacs](https://www.nuget.org/packages/Microsoft.SqlServer.Dacpacs)|Microsoft.SqlServer.Dacpacs is a set of NuGet packages containing .dacpac files for Microsoft SQL Server and Microsoft Azure SQL Databases.|
```
Comment (zijchen) [inline README.md:11]: Should we mention the Azure and Synapse variants too?

**Review by zijchen — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (dzsquared) [reply README.md]: yep, definitely should. added and merging

**Pull request merged**